sfu-config.json

```diff
@@ -1,9 +1,10 @@
-{
-    "origin": "http://localhost:8080/conferences/",
-    "edges": [
-        {
-            "sfu": "http://localhost:8081/conferences/",
-            "egress": "http://localhost:8001/api"
-        }
-    ]
+{

```

Comment:
I don't like that we are changing the format for this JSON file, why is that needed?

Reply:
On the other hand this is isolated to the reference implementation right, and then it is ok assuming no API changes has been made in `registerBroadcasterClient()`

Reply:
I updated the json to be more in line with the data recieved from a Resource Manager. The broadcasterClient has not been modified.